scripts/process.ts

```diff
@@ -137,6 +137,58 @@ let params = {
     ...SYSTEM_PARAMS
 };
 
+
+function extractTestFunctionNames(sourceCode: string): string[] {
+    // Remove single-line comments
+    sourceCode = sourceCode.replace(/\/\/[^\n]*/g, '');
+
+    // Remove multi-line comments
+    sourceCode = sourceCode.replace(/\/\*[\s\S]*?\*\//g, '');
+
+
+    const regexPatterns = [
+        /function\s+(TEST\w+)/g,
```

Comment:
It seems like:

```
function TEST_HOOK_PTR() -> ret {
    ret := LAST_FREE_SLOT()
}

function TEST_HOOK_PARAMS_OFFSET() -> ret {
    ret := sub(TEST_HOOK_PTR(), mul(5, 32))
}
```

these would be matched too, though it is not a big deal

Reply:
yes, but I'm running this only over bootloader_test.yul (and those functions are defined in another file)